### PR TITLE
Update loom from 0.145.0 to 0.147.2

### PR DESCRIPTION
**Pull request by p-linnane**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by p-linnane** [comment]
Never seen this one for `:electron_builder`.
```
audit for loom: failed
 - exception while auditing loom: Tried to load unspecified class: Time
```

**Review by samford — COMMENTED** [review]
> exception while auditing loom: Tried to load unspecified class: Time

This error comes from [`Psych#safe_load`](https://ruby-doc.org/stdlib-2.6.10/libdoc/psych/rdoc/Psych.html#method-c-safe_load) since we use [`YAML.safe_load(content)`](https://github.com/Homebrew/brew/blob/001bacee18c512657e1535dff757c8cfdaefc116/Library/Homebrew/livecheck/strategy/electron_builder.rb#L50) in `ElectronBuilder`. This `latest-mac.yml` file contains `releaseDate: 2022-12-01T02:02:46.419Z` but `#safe_load` doesn't allow a datetime to be deserialized by default, so we receive this error.

Long story short, we can fix this by modifying the related `#safe_load` call in `ElectronBuilder` to allow `Time` [in addition to the existing defaults] (i.e., `yaml = YAML.safe_load(content, permitted_classes: [Time])`). This is an easy fix, so I'll create a PR for this in a moment (edit: Homebrew/brew#14205).

**Review by bevanjkay — APPROVED** [review]
